package NYT.NCGroup.NProto;

////////////////////////////////////////////////////////////////////////////////

message TCpuAccountingStatistics
{
    required int64 user_time = 1;
    required int64 system_time = 2;
}

message TBlockIOStatistics
{
    required int64 total_sectors = 1;
    required int64 bytes_read = 2;
    required int64 bytes_written = 3;
}

////////////////////////////////////////////////////////////////////////////////
